### ISSUE TITLE
[Bug] Bar wont be at top z order

### ISSUE THREAD

**Issue by GerardPolloRebozado** [glzr-io/zebar#135]
### Describe the bug

If I open any program the program will hide the bar
The configuration is set to `  "zOrder": "top_most",`  but the base will only be visible at desktop

### Reproduction

Just installed the app, first time using it and trying to setup I just used the default starter config for glazewm and changethe zOrder and color to black

### Version number

zebar --version show 0.0.0 but I downloaded from github releases the 2.4.0

**Comment by veryard** [comment]
Hey @GerardPolloRebozado,

Can you please share the config of the default start that you changed, and if possible some pictures or even a video so I can replicate this on my end.

Thanks!

**Comment by GerardPolloRebozado** [comment]
[with-glazewm.zebar.json](https://github.com/user-attachments/files/17610189/with-glazewm.zebar.json)
[styles.css.txt](https://github.com/user-att
[with-glazewm.html.txt](https://github.com/user-attachments/files/17610194/with-glazewm.html.txt)
achments/files/17610193/styles.css.txt)
https://github.com/user-attachments/assets/0c091ac2-6d4b-424b-b74e-dc1dabe755d7
Both html and .css added the .txt extension to be able to upload to github but in my computer are with the correct extension.
I modified the originar starter template for glazeWM

**Comment by lars-berger** [comment]
> [with-glazewm.zebar.json](https://github.com/user-attachments/files/17610189/with-glazewm.zebar.json) [styles.css.txt](https://github.com/user-attachments/files/17610193/styles.css.txt) [with-glazewm.html.txt](https://github.com/user-attachments/files/17610194/with-glazewm.html.txt) https://github.com/user-attachments/assets/0c091ac2-6d4b-424b-b74e-dc1dabe755d7 Both html and .css added the .txt extension to be able to upload to github but in my computer are with the correct extension. I modified the originar starter template for glazeWM

After changing the `.zebar.json` files, the config has to be reloaded either through the system tray option or by restarting Zebar. Could you double check if the z-order is still the same after a restart?

**Comment by GerardPolloRebozado** [comment]
Yes the z-order doesnt change. I tried click in reload and also exit and opening it again but the configuration doesn't change

**Comment by whaaaley** [comment]
Ran into this today as well. I was hoping to get my Bar to overlay on top of my task bar.

**Comment by Cybernetics354** [comment]
is there any update on this issue ?

**Comment by fm-sys** [comment]
> Ran into this today as well. I was hoping to get my Bar to overlay on top of my task bar.

Just spend several hours on this one, unfortunately it seems to be just impossible :/

I was just scripting in python, not with a real Zebar code base. While the window cannot stay at top during the taskbar has focus, it's possible to get the focus again afterwards. That's as far as I could get:


```py
import tkinter as tk
import ctypes, time

user32 = ctypes.windll.user32
FindWindowExW = user32.FindWindowExW
GetForegroundWindow = user32.GetForegroundWindow

# Function to find all top-level HWNDs of a given class
def find_all_hwnds_by_class(class_name):
    hwnds = []
    hwnd = None
    while True:
        hwnd = FindWindowExW(None, hwnd, class_name, None)
        if not hwnd:
            break
        hwnds.append(hwnd)
    return hwnds

# ---------------- Tkinter Overlay ----------------
root = tk.Tk()
root.overrideredirect(True)

# Position overlay at bottom center
screen_width = root.winfo_screenwidth()
screen_height = root.winfo_screenheight()
window_width = 300
window_height = 50
x = screen_width - window_width - 10
y = screen_height - window_height - 10
root.geometry(f"{window_width}x{window_height}+{x}+{y}")

# Draw overlay
canvas = tk.Canvas(root, width=window_width, height=window_height,
                   bg='lightblue', highlightthickness=0)
canvas.pack()
canvas.create_text(window_width // 2, window_height // 2,
                   text="Overlay Example", font=("Arial", 16))

# Get all top-level Shell_TrayWnd HWNDs
taskbar_hwnds = find_all_hwnds_by_class("Shell_TrayWnd") + find_all_hwnds_by_class("Windows.UI.Core.CoreWindow")

print(taskbar_hwnds)

# Flag to avoid repeatedly forcing topmost
focus_reclaimed = False

def maintain_topmost_and_focus():
    global focus_reclaimed
    fg_hwnd = GetForegroundWindow()

    if fg_hwnd in taskbar_hwnds:
        focus_reclaimed = False
    else:
        if not focus_reclaimed:
            # Toggle topmost to stay above everything
            root.attributes('-topmost', False)
            root.attributes('-topmost', True)
            time.sleep(0.3)
            root.attributes('-topmost', False)
            root.attributes('-topmost', True)
            focus_reclaimed = True

    root.after(50, maintain_topmost_and_focus)

# Start topmost and monitoring
root.attributes('-topmost', True)
maintain_topmost_and_focus()
root.mainloop()

```

**Comment by remeyou** [comment]
Another way to solve the task bar overlay is making task bar could be click through.
On my Windows 11, a [Windhawk](https://github.com/ramensoftware/windhawk) module named "Windows 11 Taskbar Styler" provides the "Click-through taskbar" to got that.
